be/src/connector/hive_connector.cpp

```diff
@@ -424,6 +425,9 @@ Status HiveDataSource::_init_scanner(RuntimeState* state) {
     FSOptions fsOptions =
             FSOptions(hdfs_scan_node.__isset.cloud_configuration ? &hdfs_scan_node.cloud_configuration : nullptr);
 
+    // Use to print file path in be.out when coredump
+    tls_thread_status.set_scan_file_path(native_file_path);
```

Comment:
when the chunk source is schedule out from the io thread, it will not call this function when it schedule back